Fakebook.Posts/Fakebook.Posts.IntegrationTests/ClientActions/PostCreation.cs

```diff
@@ -248,5 +252,32 @@ public async Task PostAsyncComment_InvalidPost_BadRequest()
             Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
         }
 
+
+
+        [Fact]
+        public void GetNewsFeedAsyn_DtoIsNotNull_ReturnsOk()
+        {
+
+                        // Arrange
+            Mock<IPostsRepository> mockRepo = new();
+            Mock<IFollowsRepository> mockFollowRepo = new();
+            Mock<IBlobService> mockBlobService = new();
+            Mock<Microsoft.Extensions.Logging.ILogger<Fakebook.Posts.RestApi.Controllers.PostsController>> mockLoggerService = new();
+            Mock<ITimeService> mockTimeService = new();
+
+            // arrange
+            var controller = new Fakebook.Posts.RestApi.Controllers.PostsController(mockRepo.Object,mockFollowRepo.Object,mockBlobService.Object,mockLoggerService.Object,mockTimeService.Object);
+
+
+
+
+            // act
+            NewsFeedDto newsFeedDto = new NewsFeedDto();
+            newsFeedDto.Emails = new List<string> { "test@domain.com, test2@domain.com" };
+            var result = controller.GetNewsfeedAsync(newsFeedDto);
```

Comment:
bad test - not even awaiting the task to see if some exception was thrown. the return value of an async method can never be null, it's always a task.

Reply:
I wrote a more useful test